app/assets/stylesheets/application.css.scss.erb

```diff
@@ -12,4 +12,14 @@ $image-path: 'img';
 }
 .text-wrap {
   @include text-wrap-mixin;
+}
+
+.button-delete {
+  color: white !important;
+  text-decoration: none !important;
```

Comment:
This is bad. I've discussed with @karlaturcios the need for a "link-button" (or "button-link"?) style in the design system, since using links that behave as buttons is a valid solution in a case like this, where we have multiple buttons trying to perform different actions in/around a form.

Reply:
We will create a new DS component for this and come back to remove local styles. Users need the delete button on this screen ASAP.